sentry_sdk/integrations/django/views.py

```diff
@@ -23,9 +23,17 @@ def patch_views():
     # type: () -> None
 
     from django.core.handlers.base import BaseHandler
+    from django.template.response import SimpleTemplateResponse
     from sentry_sdk.integrations.django import DjangoIntegration
 
     old_make_view_atomic = BaseHandler.make_view_atomic
+    original_render = SimpleTemplateResponse.render
+
+    def wrapped_render(self):
+        # type: (SimpleTemplateResponse) -> Any
+        hub = Hub.current
+        with hub.start_span(op="django.response", description="render"):
```

Comment:
we're trying to be more consistent in our [Span Ops naming](https://develop.sentry.dev/sdk/performance/span-operations/), so can you make the op `view.response` or `view.response.render` and add it as a constant [here](https://github.com/getsentry/sentry-python/blob/e2e0de10a0614bb8fb8768757849dce584f381cf/sentry_sdk/consts.py#L73)?

Reply:
also for the description, is there any more specific information we could add instead of just `render`?

Reply:
I'm not sure on the description. Basically Django serializes the Python objects to HTML, JSON, or plain text

Reply:
I've updated the PR, ready for another review when you have a moment @sl0thentr0py